great_expectations_cloud/logging_cfg.py

```diff
@@ -54,5 +54,27 @@ def configure_logger(log_level: LogLevel, log_cfg_file: pathlib.Path | None) ->
         logging.config.dictConfig(dict_config)
         LOGGER.info(f"Configured logging from file {log_cfg_file}")
     else:
-        logging.basicConfig(level=log_level.numeric_level)
-        LOGGER.info(f"Configured logging at level {log_level}")
+        logDirectory = pathlib.Path("logs")
+        if not logDirectory.exists():
+            pathlib.Path(logDirectory).mkdir()
+
+        logger = logging.getLogger()
+        formatter = logging.Formatter(
+            "%(asctime)s | %(name)s | %(lineno)d | %(levelname)s: %(message)s"
```

Comment:
See screenshot in the description of this PR to see how this turns out. I thought it was nice to have the line number in there, too.

Reply:
Agree on including the line number :) One question: Is it customary to have the number just by itself (e.g. for easier parsing)? Or have a `line:` or some other prefix before the line number?

Reply:
I like the idea of using `line:`; that makes it easier to understand--will update, thank you!